modules/olm-updating-index-image.adoc

```diff
@@ -1,43 +1,45 @@
 // Module included in the following assemblies:
 //
 // * operators/olm-managing-custom-catalogs.adoc
+// * operators/olm-restricted-network.adoc
 
 [id="olm-updating-index-image_{context}"]
 = Updating an index image
 
-You can update an existing index image using the `opm` CLI.
+After configuring OperatorHub to use a CatalogSource that references a custom index image, cluster administrators can keep the available Operators on their cluster up to date by adding bundle images to the index image.
+
+You can update an existing index image using the `opm index add` commad.
 
 .Prerequisites
 
 * `opm` version 1.12.3+
 * `podman` version 1.4.4+
 * An index image built and pushed to a registry.
-* A CatalogSource created and applied to a cluster.
+* An existing CatalogSource referencing the index image.
 
 .Procedure
 
-. Update the existing index:
+. Update the existing index by adding bundle images:
 +
 [source,terminal]
 ----
 $ opm index add \
-    --bundles quay.io/<namespace>/another-operator:v1 \//<1>
-    --from-index quay.io/<namespace>/test-catalog:latest \//<2>
-    --tag quay.io/<namespace>/test-catalog:latest <3>
+    --bundles <registry>/<namespace>/<new_bundle_image>:<tag> \//<1>
+    --from-index <registry>/<namespace>/<existing_index_image>:<tag> \//<2>
```

Comment:
@adellape I suggest to use same image name for both "--from-index" and "--tag" so that the current index image itself is updated. So
```console
--from-index <registry>/<namespace>/<existing_index_image>:<tag>
--tag <registry>/<namespace>/<existing_index_image>:<tag>
```

Reply:
I was considering whether they might want to rename the updated image using `--tag`, but you're right, better to keep it clear here.